Adding some instructions for IntelliJ and Maven (#18)
gurkenlabs/litiengine-docs: getting-started/development-environment.md

![eclipse-download-5](/getting-started/img/eclipse_download_5.png)

### IntelliJ IDEA

> To make shure the JDK got installed correctly, open the `New Project` dialogue and expand the `JDK` dropdown menu. If you see your installed JDK in the list, you're set. If you don't, you have to click on the `Add JDK...` button. A file picker dialogue will open, choose youre JDK folder there.

Reviewer: ```suggestion
To make sure the JDK got installed correctly, open the `New Project` dialogue and expand the `JDK` dropdown menu. If you see your installed JDK in the list, you're set. If you don't, you have to click on the `Add JDK...` button. A file picker dialogue will open, choose your JDK folder there.
```

> > If you're on linux or mac, I would hightly recommend to use [SKDMAN!](https://sdkman.io/). If you installed sdk man, type `sdk install java` in your terminal. The latest recommend JDK will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage).

Reviewer: ```suggestion
> If you're on linux or mac, you can also use [SDKMAN!](https://sdkman.io/) to manage your JDK installation. If you installed sdk man, type `sdk install java` in your terminal. The latest recommend JDK will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage).
```

### IntelliJ IDEA
To make sure the JDK got installed correctly, open the `New Project` dialogue and expand the `JDK` dropdown menu. If you see your installed JDK in the list, you're set. If you don't, you have to click on the `Add JDK...` button. A file picker dialogue will open, choose your JDK folder there.

> > If you're on linux or mac, you can also use [SKDMAN!](https://sdkman.io/) to manager your JDK installation . As soon you installed sdk man, type `sdk install java` in your terminal. The latest recommend JDK will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage).

Reviewer: ```suggestion
> If you're on linux or mac, you can also use [SKDMAN!](https://sdkman.io/) to manager your JDK installation . Once you've installed sdk man, type `sdk install java` in your terminal. The latest recommend JDK will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage).
```
